src/tapis-api/files/mkdir.ts

```diff
@@ -0,0 +1,18 @@
+import { Files } from '@tapis/tapis-typescript';
+import { apiGenerator, errorDecoder } from 'tapis-api/utils';
+
+const mkdir = (
+  params: Files.MkdirOperationRequest,
```

Comment:
Instead of accepting a Files.MkdirOperationRequest, accept `systemId` and `path` as parameters and perform the wrapping of those values into a MkdirOpoerationRequest inside of the function body.

Reply:
https://github.com/tapis-project/tapis-ui/blob/main/src/tapis-api/authenticator/login.ts is a good reference